packages/inventory-compliance/src/RulesComplianceFilter.test.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import RulesComplianceFilter from './RulesComplianceFilter';
+import { IntlProvider } from 'react-intl';
+
+describe('RulesComplianceFilter component', () => {
+    const policies = [
+        {
+            refId: 'xccdf_org.ssgproject.content_profile_pci-dss',
+            name: 'PCI-DSS v3 Control Baseline for Red Hat Enterprise Linux 7'
+        },
+        {
+            refId: 'xccdf_org.ssgproject.content_profile_pci-dss',
+            name: 'PCI-DSS v3 Control Baseline for Red Hat Enterprise Linux 8'
+        }
+    ];
+
+    it('should render filter', () => {
+        const wrapper = shallow(
+            <RulesComplianceFilter availablePolicies={ policies } />
+        );
+        expect(toJson(wrapper)).toMatchSnapshot();
+    });
+
+    it('should not allow to filter if there is only one policy', () => {
+        const wrapper = shallow(
+            <RulesComplianceFilter availablePolicies={ [ policies[0] ] } />
+        );
+        const instance = wrapper.instance();
+        expect(wrapper.state('filterCategories').length).toEqual(2);
+    });
+
+    it('should allow to add and remove filters', () => {
+        const wrapper = shallow(
+            <RulesComplianceFilter availablePolicies={ policies } />
+        );
+        const instance = wrapper.instance();
+        instance.addFilter('hidePassed', true);
+        expect(wrapper.state('hidePassed')).toEqual(true);
```

Comment:
Can we limit the amount of expects per test to 2 or 3? This way if one of these fails the entire test case fails and you'd have hard time debugging what actually failed out of these 7 expects.